test/unit/uk/gov/hmrc/exports/migrations/repositories/TestObjectsBuilder.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.exports.migrations.repositories
+
+import com.mongodb.client.MongoCursor
+import com.mongodb.{ServerAddress, ServerCursor}
+import org.bson.Document
+
+object TestObjectsBuilder {
+  def buildMongoCursor(elements: Seq[Document]): MongoCursor[Document] = new MongoCursor[Document] {
+    private val iterator = elements.iterator
+
+    override def close(): Unit = ???
```

Comment:
What's all this un-implemented stuff for?

Reply:
I have not found a way to easily create `MongoCursor` to be able to return it from mocks. So I thought about mocking this as well, but in the end it turned out that in this case it is easier to create anonymous object with minimum implementation. Some of the methods are not implemented, because they are not used in tests. Besides, some of them are specific to DB operations, while I only need the iterating part of the class.

Reply:
Ok, seems reasonable.  Maybe add a comment to explain why these methods are "unfinished"?

Reply:
Can do. But I'll wait until you finish the review, so that my pushes not interfere it.

Reply:
I've finished - go for it